test/PSReadLine/project.json

```diff
@@ -13,10 +13,10 @@
     },
```

Comment:
The dependency to `Microsoft.PowerShell.PSReadLine` should be `"6.0.0-*"`. It's an oversight from the change that bounce the project versions. Would you mind update it in this PR?